index.html

```diff
@@ -6,43 +6,45 @@
 permalink: /
 ---
 
+<p>An open source license protects contributors and users. Businesses and savvy developers won’t touch a project without this protection.</p>
+
 <h2>
   <span>{</span>
   Which of the following best describes your situation?
   <span>}</span>
 </h2>
 <ul class="triptych situations clearfix">
-  <li class="whatever">
-    <a href="licenses/mit">
-      <span class="triptych-sprite three-arrows"></span>
-      <h3>I want it simple and permissive.</h3>
+  <li class="patents">
+    <a href="existing">
+      <span class="triptych-sprite lightbulb"></span>
```

Comment:
Probably a ["community"/"group" icon](https://commons.wikimedia.org/wiki/Category:People_group_icons) would work well to illustrate this concept.